From b20345ad5db13cf9a8ca8f5cb036ef526ab6693b Mon Sep 17 00:00:00 2001
From: Eric Garver <e@erig.me>
Date: Tue, 12 Dec 2017 13:57:55 -0500
Subject: [PATCH] services/high-availability: Add port 9929

TCP/UDP 9929 is used by boothd and should be opened for
high-availability.

Resolves: RHBZ 1486143
---
 config/services/high-availability.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/services/high-availability.xml b/config/services/high-availability.xml
index 002dd0e4a9d0..b6d14b793250 100644
--- a/config/services/high-availability.xml
+++ b/config/services/high-availability.xml
@@ -7,5 +7,7 @@
   <port protocol="tcp" port="5403"/>
   <port protocol="udp" port="5404"/>
   <port protocol="udp" port="5405"/>
+  <port protocol="tcp" port="9929"/>
+  <port protocol="udp" port="9929"/>
   <port protocol="tcp" port="21064"/>
 </service>
-- 
2.12.0

